apps/web/app/(app)/onboarding/OnboardingColdEmailBlocker.tsx

```
@@ -0,0 +1,20 @@
+"use client";
+
+import { useRouter, useSearchParams } from "next/navigation";
+import { ColdEmailForm } from "@/app/(app)/cold-email-blocker/ColdEmailSettings";
+
+export function OnboardingColdEmailBlocker() {
+  const router = useRouter();
+  const searchParams = useSearchParams();
+  const step = parseInt(searchParams.get("step") || "1");
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)